src/theme/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment, ReactElement, ReactNode } from 'react'
+import { ThemeProvider } from 'styled-components'
+
+import { defaultTheme as theme } from './default'
+import { GlobalStyle } from './GlobalStyle'
+
+export const Theme = ({
```

Comment:
Tady mám trochu problém přečíst, co se vlastně děje. Naformátoval to takto Prettier/eslint? Pokud ano, zvážil bych jeho nastavení :-)